src/Entity/DetectionResult.php

```diff
@@ -2,21 +2,25 @@
 
 namespace App\Entity;
 
-use ApiPlatform\Core\Annotation\ApiResource;
+use ApiPlatform\Action\NotFoundAction;
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Get;
+use ApiPlatform\Metadata\Post;
 use App\Repository\DetectionResultRepository;
 use Doctrine\ORM\Mapping as ORM;
 use Symfony\Component\Serializer\Annotation\Groups;
 
+#[ApiResource(
+    operations: [
+        new Get(status: 404, controller: NotFoundAction::class, read: false),
```

Comment:
I agree. I tried to remove it in a OpenApiFactory but it didn't work for some reason, and I didn't want to use to much time on solving it.

As for why, I just follow the manual: https://api-platform.com/docs/core/messenger/#dispatching-a-resource-through-the-message-bus

> We disabled the item operation using the NotFoundAction. A resource must have at least one item operation as it must be identified by an IRI, here the route /people/1 exists, eventhough it returns a 404 status code.

I'm not sure if the item operation can be safely omitted so I left it in, even if it seems redundant.

Reply:
Yes, I have read that and came to the same conclusion in the aarhus-mosic api and simple removed it only having the POST there (as the only one, with message, so I do not need an IRI for that entity as it is input only with message bus. If it had been a "normal" post that returned content created or something it might be needed).

But at we only return an empty 202 I think it's safe to remove and get a cleaner API.

Reply:
Removed for cleaner API